src/Common/src/System/Globalization/IdnMapping.Windows.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.Globalization
 {
-    sealed partial class IdnMapping
+    internal sealed partial class IdnMapping
```

Comment:
This change doesn't have anything to do with Collections. I'd say revert it and keep the purpose of this PR singular.